teamA/leeHyeonTae/controllers/userController.js

```diff
@@ -0,0 +1,23 @@
+const userService = require('../services/userService');
+
+const signUp = async (req, res) => {
+    try{
+        const { name, email, profile_image, password } = req.body;
+
+        if( !name || !email || !profile_image || !password ){
+            return res.status(400).json({ message : 'error! key_error of userController'});
```

Comment:
회원가입 과정에서 발생하는 오류를 클라이언트에게 알려주는 에러메시지입니다.

어떤 단계에서 에러가 발생했는지보단 어떤 에러인지 명확하게 보여주는 것이 중요합니다.

다음과 같이 수정할 수 있겠네요!
```suggestion
            return res.status(400).json({ message : 'Key Error'});
```